alchemy/light-account/Makefile

```diff
@@ -0,0 +1,14 @@
+FOUNDRY_VERSION := 1.0.0
+VERSIONS += 1.1.0
+VERSIONS += 2.0.0
+
+.PHONY: all $(VERSIONS)
+all: $(VERSIONS)
+
+$(VERSIONS):
+	@$(MAKE) -C $@ -f ../../../Makefile.project
+
+2.0.0:
+	$(MAKE) -f ../../Makefile install-foundry FOUNDRY_VERSION=nightly-0fafdc2fc714869b86655713cdcc3414262f9b46
+	$(MAKE) -C $@ -f ../../../Makefile.project
```

Comment:
We can set `FOUNDRY_DISABLE_NIGHTLY_WARNING` here to suppress warnings:

```
[
  'anvil: Warning: This is a nightly build of Foundry. It is recommended to use the latest stable version. Visit https://book.getfoundry.sh/announcements for more information. ',
  'anvil: To mute this warning set `FOUNDRY_DISABLE_NIGHTLY_WARNING` in your environment. ',
  'anvil: ',
  'anvil: '
]
```

Reply:
Alternatively, we can use version 0.3.0, which also works, and is only 1 month apart from the last working nightly.